toonz/sources/common/tstream/tstream.cpp

```diff
@@ -959,7 +957,7 @@ TIStream::TIStream(const TFilePath &fp)
 		if (check_len != out_len)
 			throw TException("corrupted file");
 
-		m_imp->m_is = new istrstream((char *)out, out_len);
+		m_imp->m_is = new istringstream((char *)out);
```

Comment:
Again, a copy of `out` is made here instead of being used as a buffer. I think `m_strbuffer` should be moved from `m_imp` to a local variable of this function.